src/demo/README.md

```diff
@@ -0,0 +1,60 @@
+## Prepare
+### Run docker-compose
+    `docker-compose -f docker-compose.yml up -d`
```

Comment:
remote quote?
https://github.com/mattrglobal/vc-authn-oidc/blob/9c456019964d1d1de0df1f481ab2e47fe179ca8f/src/demo/README.md

Reply:
ahh...line padding and markdown...
👍